src/app/pages/home/components/home-tabs.tsx

```diff
@@ -23,13 +23,16 @@ export function HomeTabs({ children }: HomeTabsProps) {
           <Tabs.Trigger data-testid="tab-assets" value={RouteUrls.Home}>
             Assets
           </Tabs.Trigger>
-          <Tabs.Trigger data-testid="tab-activity" value={`${RouteUrls.Home}${RouteUrls.Activity}`}>
-            Activity
+          <Tabs.Trigger data-testid="tab-activity" value={RouteUrls.Activity}>
+            {/* TODO - do we also want to update the route to history? */}
```

Comment:
We call it `activity-list` too so not sure important to coordinate the naming here, but we might want to in the future?

Reply:
In the design I saw the name of this tab changed to be `History` so I updated it. I'm not sure about the route name, what do you think @kyranjamie . 

Should I update the route name? Would that mess with our current stats?

Reply:
I would leave as is for now. There's been conversation about renaming in past but we can worry about that later i think

Reply:
Thanks. I'll revert to `Activity`